.github/workflows/backend-test.yml

```diff
@@ -48,9 +47,8 @@ jobs:
       - name: Install dependencies
         working-directory: backend
         run: poetry install
-
       - name: pytest
         working-directory: backend
-        run: poetry run py.test integration_tests/
+        run: poetry run pytest */integration_tests/
```

Comment:
uhm, we can either mark integration tests with something like @pytest.skip.if(INTEGRATION=False) or have them somewhere else :)

Reply:
Do you have any preference?

Reply:
something like this if it works ;) https://gist.github.com/billyshambrook/264624b039fb3caa9278

Reply:
have a look

Reply:
top! we can merge ;)